Refresh.GameServer/Endpoints/Game/Levels/LevelEndpoints.cs

```diff
@@ -0,0 +1,36 @@
+using System.Net;
+using Bunkum.HttpServer;
+using Bunkum.HttpServer.Endpoints;
+using Bunkum.HttpServer.Responses;
+using Refresh.GameServer.Database;
+using Refresh.GameServer.Types.Levels;
+using Refresh.GameServer.Types.Levels.Categories;
+using Refresh.GameServer.Types.Lists;
+using Refresh.GameServer.Types.UserData;
+
+namespace Refresh.GameServer.Endpoints.Game.Levels;
+
+public class LevelEndpoints : EndpointGroup
+{
+    // Workaround shitty routing
```

Comment:
workarounds like this should be accompanied by a `FIXME`, perhaps down on line 21 the route could be a `string?` and when its null that means theres no specific route?

Reply:
True, I should have given this a bit more thought. I can put another `GameEndpoint` parameter here to better solve this, as the router wont route to this endpoint if route is missing entirely.

This will probably require a Bunkum change as I don't think it would add a null parameter in this case.

Reply:
I added a FIXME and a link back to this thread, so I'm marking as resolved for now as this isn't too critical